### PR TITLE
Sanjo/contact prototype

### PR DESCRIPTION
**Pull request by SanjoDeundiak**

### CODE REVIEW

File: implementations/rs/ockam/src/contact/contact.rs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            vault,
+        }
+    }
+}
+
+impl Contact {
+    pub fn identifier(&self) -> &str {
+        &self.identifier
+    }
+    pub fn events(&self) -> &Vec<ContactEvent> {
```
Comment (mikelodder7) [inline implementations/rs/ockam/src/contact/contact.rs:33]:
```suggestion
    pub fn events(&self) -> &[ContactEvent] {
```

**Review by mikelodder7 — CHANGES_REQUESTED** [review]
Just one small change. Lots of similarities between contact and profile. A subsequent PR could extract the common components.

File: implementations/rs/ockam/src/contact/contact.rs (hunk 1 above)
Comment (SanjoDeundiak) [reply implementations/rs/ockam/src/contact/contact.rs:33]: @mikelodder7 I wonder why returning slice is better

**Comment by SanjoDeundiak** [comment]
> Just one small change. Lots of similarities between contact and profile. A subsequent PR could extract the common components.

I agree, but I'm tolerant to copy&pasting code if logically two pisces should be separate

File: implementations/rs/ockam/src/contact/contact.rs (hunk 1 above)
Comment (SanjoDeundiak) [reply implementations/rs/ockam/src/contact/contact.rs:33]: The only case coming up to my mind - it simplifies things when we want to use Vec vs something heapless depending on enabled features

**Review by mikelodder7 — APPROVED** [review]

**Pull request closed without merging**